apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex

```diff
@@ -90,6 +90,23 @@ defmodule OMG.Eth.EthereumHeightMonitor do
     {:noreply, state}
   end
 
+  # we need to figure out where :sslsocket is getting closed
+  # because it randomly kills this process and block getter
+  # https://github.com/omisego/elixir-omg/issues/1541
```

Comment:
slightly encourage to link this one instead: https://github.com/omisego/elixir-omg/issues/1529